config/webhook/manifests.yaml

```diff
@@ -8,6 +7,7 @@ metadata:
 webhooks:
 - admissionReviewVersions:
   - v1
+  - v1beta1
```

Comment:
I'm curious as to why some of the webhooks here have both `v1` and `v1beta1` for `admissionReviewVersions` while others only have `v1`.  Is this because some have been around longer and need to be backwards compatible, whereas others are newer?  It looks like it varies for both mutating and validating webhooks.

Reply:
check https://github.com/openstack-k8s-operators/osp-director-operator/pull/555

Reply:
It would seem that #555 has only `v1` for everything.  That's why I'm a bit puzzled by the inclusion of both `v1` and `v1beta1` in places here.

Reply:
yes, thats from the changes we landed in parallel which added new webhooks. we should probably first land https://github.com/openstack-k8s-operators/osp-director-operator/pull/555 and then this here

Reply:
Exactly. I rebased this on Martin's fix so we should be good